Storage/SQL/SQLiteHistory.swift

```diff
@@ -157,6 +169,24 @@ open class SQLiteHistory {
         let args: Args = []
         return db.runQueryConcurrently(sql, args: args, factory: SQLiteHistory.iconHistoryColumnFactory)
     }
+
+    public func countVisits(callback: @escaping (Int?) -> Void) {
```

Comment:
@tarikeshaq is this a method we plan on keeping for long or the idea is to get rid of it once you have the required data?

Reply:
The plan is to keep the data for long so we can monitor the sizes of databases and compare this implementation and the application-services one. But we will be getting rid of this method when we swap to using application services history

That said, also with the application services history we will want to keep track of the number of visits, but that will probably happen in Rust as opposed to here in the app

Reply:
@tarikeshaq good to merge, let me know if you have proper permission

Reply:
@nbhasin2 thanks a ton!!! unfortunely I don't :( Is there a way I can get the permission to land a PR once it's approved?

Reply:
Merged!!